### PR TITLE
New line in readme

### PR DESCRIPTION
**Pull request by kunGup**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # Repo-to-learn-PR
 This repository is used to learn the pull request practise of github.
+New line added by kunbeg.
```
Comment (kunbeg2) [inline README.md]: Please change 'added' to 'contributed'

**Review by kunbeg2 — APPROVED** [review]
Read the comment

File: README.md (hunk 1 above)
Comment (kunGup) [reply README.md]: ok sir I will do this in the new commit

**Comment by kunbeg2** [comment]
Ok I am merging the changes.

**Pull request merged**